### PR TITLE
refactor: install cuda keyring package

### PR DESCRIPTION
**Pull request by wep21**
## Description

https://developer.nvidia.com/blog/updating-the-cuda-linux-gpg-repository-key/

```
❯ dpkg -L cuda-keyring
/.
/etc
/etc/apt
/etc/apt/preferences.d
/etc/apt/preferences.d/cuda-repository-pin-600
/etc/apt/sources.list.d
/etc/apt/sources.list.d/cuda-ubuntu2004-x86_64.list
/usr
/usr/share
/usr/share/doc
/usr/share/doc/cuda-keyring
/usr/share/doc/cuda-keyring/changelog.Debian.gz
/usr/share/keyrings
/usr/share/keyrings/cuda-archive-keyring.gpg
```

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: ansible/roles/cuda/tasks/main.yaml [hunk 1]
```diff
@@ -8,23 +8,9 @@
...
-  ansible.builtin.apt_key:
-    url: https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/{{ cuda_architecture.stdout }}/3bf863cc.pub
-
-- name: Add CUDA repository into sources.list
-  become: true
-  ansible.builtin.apt_repository:
-    repo: deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/{{ cuda_architecture.stdout }}/ /
-    filename: cuda
-    state: present
+- name: Install cuda keyring
```
Comment (kenji-miyake) [inline ansible/roles/cuda/tasks/main.yaml]: [style]

```suggestion
- name: Install CUDA keyring
```

File: ansible/roles/cuda/tasks/main.yaml [hunk 2]
```diff
@@ -8,23 +8,9 @@
...
-  ansible.builtin.get_url:
-    url: https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/{{ cuda_architecture.stdout }}/cuda-ubuntu2004.pin
-    dest: /etc/apt/preferences.d/cuda-repository-pin-600
-
-- name: Add NVIDIA apt key
-  become: true
-  ansible.builtin.apt_key:
-    url: https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/{{ cuda_architecture.stdout }}/3bf863cc.pub
-
-- name: Add CUDA repository into sources.list
```
Comment (kenji-miyake) [inline ansible/roles/cuda/tasks/main.yaml:22]: Could you run the `build` workflow? I'm not sure if we can delete this task.

Comment (wep21) [reply ansible/roles/cuda/tasks/main.yaml:22]: https://github.com/wep21/autoware/runs/6269396467?check_suite_focus=true

**Review by kenji-miyake — APPROVED** [review]

**Comment by wep21** [comment]
@kenji-miyake Could you merge this PR because I don' have a write access right to this repository?

**Pull request merged**